flake8_html/plugin.py

```diff
@@ -13,6 +13,8 @@
 import codecs
 import datetime
 import pkgutil
+import configparser
```

Comment:
I absolutely agree. I totally overlooked flake8's native config parsing. It's way neater.